src/controller/BackPostController.php

```diff
@@ -91,7 +91,8 @@ public function editPost(string $idPost, PostModel $postModel)
 
 		// The form is not submitted, posting the post form
 		if (count($_POST) === 0) {
-			echo $this->twig->render('insert_post.twig', ['SESSION' => $_SESSION, 'post' => $post]);
+			// echo $this->twig->render('insert_post.twig', ['SESSION' => $_SESSION, 'post' => $post]);
+			$this->render('insert_post.twig', [], [], $_SESSION, $post);
```

Comment:
idem pour session

Reply:
Modif OK : 
$this->render('insert_post.twig', $_SESSION, [], [], $post);